src/main/java/edu/wpi/first/math/Drake.java

```diff
@@ -50,7 +50,7 @@ public static SimpleMatrix discreteAlgebraicRiccatiEquation(
           Matrix B,
           Matrix Q,
           Matrix R) {
-    return discreteAlgebraicRiccatiEquation(A.getStorage(), B.getStorage(), Q.getStorage(),
-            R.getStorage());
+    return discreteAlgebraicRiccatiEquation(A.m_storage, B.m_storage, Q.m_storage,
```

Comment:
Why? Properties are much safer, and not a big fan of package private.

Reply:
We wanted to not expose `SimpleMatrix` to the user as much as possible: it was being used unnecessarily in the API and made some thing really verbose. That is why `getStorage()` was removed and the storage became package private. We can add back `getStorage()` as package private, but it doesn't need to be public.

Reply:
While I agree that it doesn't _need_ to be package-private, I'd still like to expose the underlying EJML matrix to teams in case they want to perform an operation that our `Matrix` wrapper doesn't yet support.